server/protocols/jmap/src/main/java/org/apache/james/jmap/DefaultMailboxesProvisioningFilter.java

```diff
@@ -57,9 +61,11 @@ public void init(FilterConfig filterConfig) throws ServletException {
     
     @Override
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
+        TimeMetric timeMetric = metricFactory.timer("JMAP-mailboxes-provisioning");
         Optional<MailboxSession> session = Optional.ofNullable((MailboxSession)request.getAttribute(AuthenticationFilter.MAILBOX_SESSION));
         session.ifPresent(this::createMailboxesIfNeeded);
         chain.doFilter(request, response);
+        timeMetric.elapseTimeInMs();
```

Comment:
is it expected to put the elapseTimeInMs call after chain.doFilter? Then you are mesuring other filters too, no?